frontend/src/Editor/Components/Button.jsx

```diff
@@ -21,7 +21,9 @@ export const Button = function Button({
   const backgroundColor = component.definition.styles.backgroundColor.value;
   const color = component.definition.styles.textColor.value;
   const widgetVisibility = component.definition.styles?.visibility?.value || true;
```

Comment:
`false || true` will always return true 
```suggestion
  const widgetVisibility = component.definition.styles?.visibility?.value ?? true;
```

Reply:
yes, I missed it, will fix it.